src/app/infra/views/sensors_records_views/last_sensors_record_page_view.py

```diff
@@ -2,6 +2,8 @@
 
 from infra.repositories import SensorRecordsRepository
 
+
+
 sensors_records_repo = SensorRecordsRepository()
```

Comment:
sensors_records_repository already exitst in __init__.py of repositories folder. Just import it and use it in the view.